install.sh

```diff
@@ -1028,6 +1029,7 @@ then
     plural="s"
   fi
   echo "- Run these commands in your terminal to add the non-default Git remote${plural} for ${non_default_repos}:"
+  echo "    echo '# Set PATH, MANPATH, etc., for Homebrew.' >> ${shell_profile}"
   printf "    echo '%s' >> ${shell_profile}\n" "${additional_shellenv_commands[@]}"
   printf "    %s\n" "${additional_shellenv_commands[@]}"
```

Comment:
Would be nice to be more consistent about usage between `printf` and `echo`. `printf` should be more portable, IIUC.

Reply:
I can try pushing that change, but at least on my local system:
```
printf "    echo '# Set PATH, MANPATH, etc., for Homebrew.' >> ${shell_profile}"
```
triggers `shellcheck` warning SC2059 (`Don't use variables in the printf format string. Use printf '..%s..' "$foo".`).

Reply:
Yea, we can leave it be.

Reply:
Yup, CI complains about it too, will adjust all the `printf`s accordingly.